sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/StorageAccountInfo.java

```diff
@@ -36,4 +51,11 @@ public SkuName getSkuName() {
     public AccountKind getAccountKind() {
         return accountKind;
     }
+
+    /**
+     * @return whether hierarchical namespace is enabled on the account.
```

Comment:
nit: Include a Javadoc description, even if it copies the return Javadoc. In Java 18+ missing descriptions are a Javadoc warning.